src/main/java/ccetl/event/listeners/LambdaListener.java

```diff
@@ -57,9 +57,13 @@ public LambdaListener(Class<? super E> target, @Nullable Predicate<E> filter, Co
     public LambdaListener(Class<? super E> target, Consumer<E> listener, byte priority) {
         this(target, null, listener, priority);
     }
-
+    
     // Additional constructors for convenience...
 
+    public LambdaListener(Class<? super E> target, byte priority, Consumer<E> listener) {
```

Comment:
You should rework all other listeners to this pattern. To avoid inconsistency. Right now this is just a duplicate and doesn't follow the overall style. If you rework all constructors to mach this pattern, please mark the other constructors as <code>@depreaced</code> or remove them. I'd like to keep overall consistency in this system.